module/Accounting/src/Accounting/Controller/AccountsController.php

```diff
@@ -69,4 +90,31 @@ protected function getResourceOptions() {
 		return self::$resourceOptions;
 	}
 	
+	public function setEventManager(EventManagerInterface $events)
```

Comment:
Avevamo concordato di creare una classe dedicata alla gestione di tutte le dipendenze dall'organizzazione, perché hai replicato il codice in tutti i controller?